apps/app/app/index.tsx

```
@@ -38,8 +45,7 @@ function Page() {
          * - ios simulator 사용 시: `http://localhost:3000`
          * - 실기기 사용 시: LAN IP 주소 ex) `http://192.0.0.1:3000`
          */
-        // TEMP: URI env에 등록하여 사용 예정
-        source={{ uri: 'http://localhost:3000' }}
+        source={isSynced ? { uri: process.env.EXPO_PUBLIC_WEB_URL ?? 'http://localhost:3000' } : undefined}
```

ESLint

[error] Unexpected use of undefined.

(no-undefined)